import.py

```diff
@@ -199,6 +199,8 @@ def xml_to_entry(xml, confkey, entry_type, fields, short_year):
     for e in elt:
         if e.tag == "author":
             authors.append(clean_author(unicode(e.text)))
+        elif e.tag == "editor" and entry_type == "Book":
```

Comment:
Now that I rethink about it, this might not be the good way of doing it, because in that case it means that the book includes several `@Incollection` that should also be added, and the editor might not be an author.